### PR TITLE
Adding placement prop to TextField component

### PR DESCRIPTION
**Pull request by barboaron**
# ✨ Pull Request

### 💁 Description

...

### 💭 Motivation

...

### ☝️ TODOs 

...

### 👀  PR is ready for review 

- [x] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_597/

File: src/components/TextField/TextField.tsx [hunk 1]
```diff
@@ -43,6 +44,8 @@ export interface TPATextFieldProps extends TPAComponentProps {
   clearButtonAriaLabel?: string;
   /** Identifies the element that labels the clear button element. Optional. */
   clearButtonAriaLabelledby?: string;
+  /** Placement for Popover. Optional. */
+  placement?: Placement;
```
Comment (sivanharel) [inline src/components/TextField/TextField.tsx:48]: Could you rename it for something more specific? It is not clear that it is only the placement of the error icon popover.

```suggestion
  errorTooltipPlacement?: Placement;
```

Comment (sivanharel) [reply src/components/TextField/TextField.tsx:48]: And maybe you could add it right after the `errorTooltipMaxWidth` prop :)

**Review by sivanharel — COMMENTED** [review]
Thanks for contributing:)
I added a very tiny comment regarding the name of the prop.

File: src/components/TextField/TextField.tsx (hunk 1 above)
Comment (barboaron) [reply src/components/TextField/TextField.tsx]: done :)

**Review by sivanharel — APPROVED** [review]

**Pull request merged**